doc/src/manual/modules.md

```diff
@@ -325,7 +325,30 @@ include(p) = Base.include(Mod, p)
 end
 ```
 
-If even `Core` is not wanted, a module that imports nothing and defines no names at all can be defined with `Module(:YourNameHere, false, false)` and code can be evaluated into it with [`@eval`](@ref) or [`Core.eval`](@ref).
+If even `Core` is not wanted, a module that imports nothing and defines no names at all can be defined with `Module(:YourNameHere, false, false)` and code can be evaluated into it with [`@eval`](@ref) or [`Core.eval`](@ref):
+```julia
+julia> arithmetic = Module(:arithmetic, false, false)
+Main.arithmetic
+
+julia> @eval arithmetic add(x, y) = $(+)(x, y)
+add (generic function with 1 method)
+
+julia> @doc """
```

Comment:
`@doc` here seems somewhat random since that isn't introduced anywhere else on this page.

Reply:
If that's the case, then should the `@doc` be put into `@ref` so users can click on it to get information on what the `@doc` says?

Reply:
> If that's the case, then should the `@doc` be put into `@ref` so users can click on it to get information on what the `@doc` says?

I've updated the examples. I think its concise now. Take a look @Keno